composer/optim/scheduler.py

```diff
@@ -537,19 +537,25 @@ class MultiStepWithWarmupScheduler(ComposerScheduler):
         rate multiplier until the warmup has completed.
 
     .. warning::
```

Comment:
nit: Thoughts about making it a `.. note::`? (Same comments for other schedulers) IMO warning feels a bit strong?

Reply:
It was warning before these changes right? Before the option to scale warmup it was very important to know how SSR changed the schedule for correctness. I feel like that is still true.

Reply:
Correct, it was warning before, so agree with @codestar12 to keep this as warning.